### ISSUE TITLE
TypeError: this.eWidget is null

### ISSUE THREAD

**Issue by mutahhar-ahmad** [jojoee/add2calendar#9]
Hi @jojoee @YoKoGFX I'm getting the following error:

![image](https://user-images.githubusercontent.com/70900865/102623514-6d3a3b80-4164-11eb-91c8-b619f3f43853.png)

**(Tried using Node v 12.18.2 and 11.15.0)**
Code(React JS): 
```
import * as Add2Calendar from "add2calendar"
import './App.css';

function App() {

  const AddToCalendar = () => {
    var singleEventArgs = {
      title       : 'Add2Calendar plugin event',
      start       : 'December 19, 2020 10:30',
      end         : 'December 19, 2020 10:50',
      location    : 'Bangkok, Thailand',
      description : 'Event description'
    };
    var singleEvent = new Add2Calendar(singleEventArgs);
    singleEvent.createWidget('#single-normal');
  }

  return (
    <div className="App">
    {/* <button onClick={AddToCalendar}>Click me</button> */}
    {AddToCalendar()}
    </div>
  );
}

export default App;

```

**Comment by jojoee** [comment]
@mutahhar-ahmad, thank you for reporting, let me check by today.

**Comment by jojoee** [comment]
@mutahhar-ahmad you're right, the plugin did not support it. So I made an update and released it in v1.1.5.

Please try
```javascript
// Example usage (React.js)

import * as Add2Calendar from "add2calendar"
import React from 'react'
import 'add2calendar/css/add2calendar.css'

class Add2CalendarComponent extends React.Component {
  componentDidMount () {
    const singleEvent = new Add2Calendar({
      title: 'Add2Calendar plugin event',
      start: 'December 19, 2020 10:30',
      end: 'December 19, 2020 10:50',
      location: 'Bangkok, Thailand',
      description: 'Event description'
    })
    singleEvent.createWidget('#single-normal')
  }

  render() {
    return (
      <div id="single-normal"></div>
    );
  }
}

function App() {
  return (
    <div className="App">
      <Add2CalendarComponent />
    </div>
  );
}

export default App;
```
